frontend/components/Step/Step.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from "react";
+import { FormContext } from "../Overlay/CreateProject.js";
+import { Files, Success, SetPrice, VerifyHumanity, Submit } from "../Forms";
+
+function Step() {
+  const { activeStepIndex, formData, setFormData } = useContext(FormContext);
+  let stepContent;
+  switch (activeStepIndex) {
+    case 0:
+      stepContent = <Files formData={formData} setFormData={setFormData} />;
+      break;
+    case 1:
+      stepContent = <SetPrice formData={formData} setFormData={setFormData} />;
+      break;
+    case 2:
+      stepContent = (
+        <VerifyHumanity formData={formData} setFormData={setFormData} />
+      );
+      break;
```

Comment:
I think we can remove this verify humanity check for now. Later on we can add it back using world coin or captcha